test/e2e/Deploy.t.sol

```diff
@@ -184,12 +452,30 @@ contract E2EDeploymentGoerliTest is DeployGoerli, CommonDeploymentTest {
   function setupPostEnvironment() public override(DeployGoerli, Deploy) {
     super.setupPostEnvironment();
   }
+
+  function test_stones_wbtc_oracle() public {
```

Comment:
Shouldn't `test_system_coin_oracle()` and `test_stones_wbtc_oracle()` be included in `E2EOracles.t.sol`?

Reply:
hmmm but the point was testing the correct deployment of the setup, in production we have the hai/weth univ3 pool + chainlink, and in goerli we use stn/btc to have a uniV3 pool as an oracle